src/screens/hardware-wallets/PairHardwareWalletAgainSheet.tsx

```diff
@@ -0,0 +1,145 @@
+import * as i18n from '@/languages';
+import React from 'react';
+import { Box, Inline, Inset, Stack, Text } from '@/design-system';
+import { ImgixImage } from '@/components/images';
+import ledgerNano from '@/assets/ledger-nano.png';
+import {
+  ledgerNanoHeight,
+  ledgerNanoWidth,
+  scaleFactor,
+} from '@/components/hardware-wallets/NanoXDeviceAnimation';
+import { Source } from 'react-native-fast-image';
+import Animated, {
+  useAnimatedStyle,
+  useDerivedValue,
+  withDelay,
+  withRepeat,
+  withSequence,
+  withTiming,
+} from 'react-native-reanimated';
+import { useDimensions } from '@/hooks';
+import { CancelButton } from '@/components/hardware-wallets/CancelButton';
+import gridDotsLight from '@/assets/dot-grid-light.png';
+import gridDotsDark from '@/assets/dot-grid-dark.png';
+import { useTheme } from '@/theme';
+import { IS_IOS } from '@/env';
+import { Layout } from '@/components/hardware-wallets/Layout';
+import { TRANSLATIONS } from '@/components/hardware-wallets/constants';
+
+const INDICATOR_SIZE = 7;
+
+export const PairHardwareWalletAgainSheet = () => {
+  const { width: deviceWidth } = useDimensions();
+  const { isDarkMode } = useTheme();
+  const connected = false;
+  const gridDotsSize = deviceWidth * scaleFactor;
+
+  const indicatorOpacity = useDerivedValue(() =>
+    withRepeat(
+      withSequence(
+        withDelay(1000, withTiming(0)),
+        withDelay(1000, withTiming(1))
+      ),
+      -1
+    )
+  );
+
+  const indicatorAnimation = useAnimatedStyle(() => ({
+    opacity: indicatorOpacity.value,
+  }));
+
+  return (
+    <Layout>
+      <Box style={{ zIndex: 1 }}>
+        <Inset horizontal="36px">
+          <Stack alignHorizontal="center" space="20px">
+            <Text align="center" color="label" weight="bold" size="26pt">
+              {i18n.t(TRANSLATIONS.looking_for_devices)}
```

Comment:
I wonder if we can change the way we export the translations to make it always be named the same. I see you use the pattern where you attach it to TRANSLATIONS constant and I use the regular i18n.l. The same goes for the import which is import * as i18n from...
It is not very friendly for editors to automatically import it